tests/Integration/WorkloadManagementSystem/Test_ParametricJob.py

```diff
@@ -0,0 +1,135 @@
+""" This is a test of the parametric job generation tools
+"""
+
+import unittest
+
+from DIRAC.Core.Base.Script import parseCommandLine
+parseCommandLine()
```

Comment:
Ah it is there :) Sorry, ignore my comment (unless you really wanted a unit test)

But if this needs to be an integration tests, than what configuration is needed for being run?